plugin/pkg/scheduler/algorithm/priorities/spreading.go

```diff
@@ -22,28 +22,42 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/plugin/pkg/scheduler/algorithm"
 )
 
-type ServiceSpread struct {
-	serviceLister algorithm.ServiceLister
+type JobSpread struct {
```

Comment:
We don't have a job abstraction in Kubernetes, at least not at this level of the stack, so I would suggest a different term than job. How about just Spread instead of JobSpread?